### PR TITLE
Updated the bundle configuration articles to Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: bundles/configuration.rst [hunk 1]
```diff
@@ -46,13 +47,13 @@ as integration of other related components:
 .. sidebar:: Using Parameters to Configure your Bundle
 
     If you don't have plans to share your bundle between projects, it doesn't
```
Comment (wouterj) [inline bundles/configuration.rst]: Then don't create a bundle...?

**Review by wouterj — APPROVED** [review]
You're on a great update streak!

File: bundles/configuration.rst (hunk 1 above)
Comment (weaverryan) [reply bundles/configuration.rst]: Haha, yea, I agree with Wouter. We can still say this is optional, of course, but if you're reading this, then you *do* want to share your bundle between projects :)

**Review by weaverryan — APPROVED** [review]

**Comment by weaverryan** [comment]
One minor comment + fix the conflict. Then we can merge!

File: bundles/configuration.rst (hunk 1 above)
Comment (javiereguiluz) [reply bundles/configuration.rst]: You are completely right! This only made sense in the past, where bundles could be non-shareable ... but that's no longer the case. Removed the entire sidebar!

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**